.github/workflows/pr.yml

```diff
@@ -0,0 +1,62 @@
+name: Tests and benchmarks
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  bench:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Update Rust
+        run: |
+          rustup override set stable
+          rustup update stable
+      - name: Install RV64E toolchain
+        run: |
+          wget https://github.com/paritytech/rustc-rv32e-toolchain/releases/download/v1.1.0/rust-rve-nightly-2024-01-05-x86_64-unknown-linux-gnu.tar.zst -O rust-rve-nightly.tar.zst
+          tar --zstd -xf rust-rve-nightly.tar.zst
+          mv rve-nightly ~/.rustup/toolchains/
+          rm rust-rve-nightly.tar.zst
+      - name: Install cargo contract
+        run: cargo install --git https://github.com/paritytech/cargo-contract.git --branch at/riscv --force --locked
```

Comment:
I think this is unnecessary...the `ink` build calls directly into the `contract-build` crate

Reply:
https://github.com/ascjones/schlau/blob/8920b85b2111f769c52be5877f18ec1239929546/src/ink.rs#L164

Reply:
True that, and other than that it's probably not required for now